docs/topics/basic-types.md

```diff
@@ -325,8 +326,17 @@ Same as for primitives, each of unsigned type has corresponding type that repres
 
 Same as for signed integer arrays, they provide similar API to `Array` class without boxing overhead. 
 
-Also, [ranges and progressions](ranges.md) are supported for `UInt` and `ULong` by classes `UIntRange`,
-`UIntProgression`, `ULongRange`, and `ULongProgression`. 
+When using unsigned arrays, a warning will be reported, indicating that this feature has not been released to stable.
```

Comment:
Seems too complicated, mb smth like (can be rephrased): "When you use unsigned arrays, you'll get a warning that indicates that this feature is not stable"?

Reply:
Thanks. Rephased and removed the detailed instruction about opt-in.